bluesky/callbacks/__init__.py

```diff
@@ -8,7 +8,6 @@
                    get_obj_fields, CollectThenCompute, LiveTable)
 from .fitting import LiveFit
 
-# for back-compatibility import deprecated plotting callbacks here
-# TODO remove from here and also from core.py
-from .core import (LiveScatter, LivePlot, LiveGrid,
-                   LiveFitPlot, LiveRaster, LiveMesh)
+# TODO: remove later as it's deprecated
```

Comment:
I am OK with supporting these top-level imports forever. The important thing was not importing matplotlib at the top level, and that problem is solved.

Reply:
It's addressed in https://github.com/NSLS-II/bluesky/pull/1133/commits/2f491f0bb725ce429502eefda52d376fc848e75a. Thanks for the review!